CMakeLists.txt

```diff
@@ -6,12 +6,15 @@
 #The full license is in the file LICENSE, distributed with this software. 
 #############################################################################
 
-cmake_minimum_required(VERSION 3.24)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
THis is a very old deprecated version of cmake. What was the reason for this downgrade?